api/src/main/java/org/apache/iceberg/IncrementalAppendScan.java

```diff
@@ -20,6 +20,8 @@
 
 package org.apache.iceberg;
 
+import org.apache.iceberg.io.CloseableIterable;
+
 /**
  * API for configuring an incremental table scan for appends only snapshots
```

Comment:
Logically, incrementalAppendScan is a subset of ChangeScan. Can we make incrementalAppendScan extend from ChangeScan?

Reply:
the hierarchy should be the other way around. `ChangeScan` extends `IncrementalAppendScan` as change scan is the more complex superset.